### ISSUE TITLE
Move to spock

### ISSUE THREAD

**Issue by eddumelendez** [eddumelendez/scc-payment#3]
Once the project is complete we can create an additional branch to show spock integration

**Comment by OlgaMaciaszek** [comment]
Changes ready: https://github.com/eddumelendez/scc-payment/pull/5

**Comment by eddumelendez** [comment]
Closing in favour of PR #5
